src/sprite_store.h

```diff
@@ -516,11 +521,25 @@ struct GuiSprites {
 	ImageData *weather[WTP_COUNT];          ///< Weather sprites.
 	ImageData *lights_rog[4];               ///< Red/orange/green light. @see LightRedOrangeGreen
 	ImageData *lights_rg[3];                ///< Red/green light. @see LightRedGreen
+
 	ImageData *message_goto;                ///< Inbox button go to location sprite.
 	ImageData *message_park;                ///< Inbox button park management sprite.
 	ImageData *message_guest;               ///< Inbox button guest window sprite.
 	ImageData *message_ride;                ///< Inbox button ride instance window sprite.
 	ImageData *message_ride_type;           ///< Inbox button ride select gui sprite.
+
+	ImageData *toolbar_main;                ///< Toolbar button main menu gui sprite.
+	ImageData *toolbar_speed;               ///< Toolbar button speed menu gui sprite.
+	ImageData *toolbar_path;                ///< Toolbar button path build gui sprite.
+	ImageData *toolbar_ride;                ///< Toolbar button ride select gui sprite.
+	ImageData *toolbar_fence;               ///< Toolbar button fence build gui sprite.
+	ImageData *toolbar_scenery;             ///< Toolbar button scenery build gui sprite.
+	ImageData *toolbar_terrain;             ///< Toolbar button terraform gui sprite.
+	ImageData *toolbar_staff;               ///< Toolbar button staff gui sprite.
+	ImageData *toolbar_inbox;               ///< Toolbar button inbox gui sprite.
+	ImageData *toolbar_finances;            ///< Toolbar button finances gui sprite.
+	ImageData *toolbar_objects;             ///< Toolbar button path objects gui sprite.
```

Comment:
Did you consider making an array of these sprites? I would think you get the same order of these things everywhere, so you may be able to exploit that in a nice `for` loop at some spots in the code.

Reply:
or a computed index, of course.